recipe/meta.yaml

```diff
@@ -13,20 +13,21 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  number: 1
+  number: 2
   skip: true  # [win or py<35]
-  script: "{{ PYTHON }} setup.py install --offline --single-version-externally-managed --record=record.txt"
```

Comment:
keep this one, this project still relies on a git submodule and that does require the ``--offline`` from here. The whole astropy ecosystem will be refactoring that helpers, but it's not there yet.